src/diffusers/models/autoencoders/autoencoder_kl.py

```diff
@@ -31,10 +31,11 @@
 )
 from ..modeling_outputs import AutoencoderKLOutput
 from ..modeling_utils import ModelMixin
+from ...loaders import PeftAdapterMixin
 from .vae import Decoder, DecoderOutput, DiagonalGaussianDistribution, Encoder
 
 
-class AutoencoderKL(ModelMixin, ConfigMixin, FromOriginalModelMixin):
+class AutoencoderKL(ModelMixin, ConfigMixin, FromOriginalModelMixin, PeftAdapterMixin):
```

Comment:
Could we also add a test for this in https://github.com/huggingface/diffusers/blob/main/tests/models/autoencoders/test_models_vae.py?

Reply:
Done, let me know if more/different is needed.